ci/prepare.sh

```diff
@@ -7,6 +7,22 @@ set -o xtrace
 # Read variables
 source ovn-scale.conf
 
+# Determine local ip of this system.
+# First, try eth0. Then, try bond0. Lastly, use the address
+# of the interface that is used by the default route.
+LOCALIP=$(ip addr show dev eth0 | grep -m 1 'inet ' | cut -d " " -f 6 | cut -d "/" -f 1)
+if [ "$LOCALIP" == "" ] ; then
+    # Try bond0
+    LOCALIP=$(ip addr show dev bond0 | grep -m 1 'inet ' | cut -d " " -f 6 | cut -d "/" -f 1)
+fi
+if [ "$LOCALIP" == "" ] ; then
+    # Try to use interface used in default route
+    PHYS_DEV=$(ip route list match 0.0.0.0/0 | grep -oP "(?<=dev )[^\s]*(?=\s)")
```

Comment:
let's put the -m1 here too!
ip route list match 0.0.0.0/0 | grep -m 1 -oP "(?<=dev )[^\s]*(?=\s)"